snooty.toml

```diff
@@ -2,7 +2,11 @@ name = "visual-studio-extension"
 title = "Visual Studio Extension"
 
 [constants]
-version = 1.0
-full-version = "{+version+}.0"
-name = "MongoDB Visual Studio Extension"
-home-page = "https://github.com/BorisDog/mongo-csharp-analyzer"
\ No newline at end of file
+product = "MongoDB Analyzer"
+product-home-page = "https://github.com/BorisDog/mongo-csharp-analyzer"
```

Comment:
I guess we should point to the main repo https://github.com/mongodb/mongo-csharp-analyzer?

Reply:
Updated to point to the main repo!